spec/index.bs

```diff
@@ -2210,20 +2180,125 @@ enum GPUTextureAspect {
+                            - |descriptor|.{{GPUTextureViewDescriptor/baseArrayLayer}} +
+                                |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is &le;
+                                |arrayCount|.
+                            - |descriptor|.{{GPUTextureViewDescriptor/format}} is |this|.{{GPUTexture/[[format]]}}
+                                or is a [=compatible texture format=] with |this|.{{GPUTexture/[[format]]}}.
+                            - If |descriptor|.{{GPUTextureViewDescriptor/dimension}} is
+                                <dl class="switch">
+                                    : {{GPUTextureViewDimension/"1d"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"1d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `1`.
+
+                                    : {{GPUTextureViewDimension/"2d"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `1`.
+
+                                    : {{GPUTextureViewDimension/"2d-array"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
+
+                                    : {{GPUTextureViewDimension/"cube"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `6`.
+                                    :: |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/width=] is
+                                        |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/height=].
+
+                                    : {{GPUTextureViewDimension/"cube-array"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is a multiple of `6`.
+                                    :: |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/width=] is
+                                        |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/height=].
+
+                                    : {{GPUTextureViewDimension/"3d"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"3d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `1`.
+                                </dl>
+                        </div>
+
+                        Then:
+                            1. Generate a {{GPUValidationError}} in the current scope with appropriate error message.
+                            1. Create a new [=invalid=] {{GPUTextureView}} and return the result.
```

Comment:
nit: "Return" stops the algorithm, so best to put it at the beginning of the line. "Return a new invalid GPUTextureView"